src/main/java/com/butbetter/applicationServices/Calculator.java

```diff
@@ -0,0 +1,51 @@
+package com.butbetter.applicationServices;
+
+import com.butbetter.applicationServices.model.CalcApiResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class Calculator implements CalculatorService {
+
+    private String calcApiUrlStart = "https://caluationapi.herokuapp.com/calc/v1/VAT?price=";
```

Comment:
I think for better readability you could write it e.g: 
```java

private String localhost = "http://localhost:8080"
private String heroku = "https://caluationapi.herokuapp.com"
private String pathOfCalutation = "/calc/v1/VAT?"
private String priceInput = "price="
private String percentInput = "&percent="
````

and than build it together